### ISSUE TITLE
About virtualization operating system

### ISSUE THREAD

**Issue by Ran-Xing** [DrDonk/DSX#4]
Hey, brother, unfortunately to see you say goodbye to ESXi. Which platform do you move to? I have used the PROXMOX operating system for so long. I think it is not a good virtualization system. ESXi is still the first choice. You can be regarded as a hero in virtualization.

**Comment by DrDonk** [comment]
Hi

I've retired from real world software development after 35 years of everything from IBM mainframe systems programming to being the CTO of a company. So my wife says it's time to do other things! Can't argue with her now we're both retired.

I've moved off Intel and I am using Apple Silicon kit at home, plus some other ARM stuff such as Raspberry PI & PICO. I was lucky enough to actually program on the original ARM1 dev kit back in 1986 and so looking forward to getting familiar with current ARM systems. So VMware really isn't much use to me now.

However I am still very interested in virtualisation of macOS and am looking at:

QEMU - https://qemu.org
UTM - https://utm.app
tart - https://tart.run
VirtualBuddy - https://github.com/insidegui/VirtualBuddy

I have a good friend who is using PROMOX on INtel and I think he's had macOS running on it.

So I'm still hacking but on different platform.

All the best

Dave
